### DISCUSSION TITLE
npm install for our purchased rappid code?

### DISCUSSION THREAD

**Discussion by elbashandy** [clientIO/joint#1834] [Q&A]
I am trying to modularize the javascript module that I am working on. Currently, we have a minified version of the rappid.min.js source code that we have purchased before. Is there a clean way to have it installed through a package manager like npm? Otherwise, I can include the source code but the problem is that it requires a bunch of packages like lodash, backbone, and jquery for which we also have downloaded the source code for rappid to work.

**Comment by kumilingus** [comment]
The JointJS+ package contains `rappid.tgz` file (`build/package/rappid.tgz`) which could be installed via `npm`/`yarn` by adding it to `package.json` as shown e.g. here: https://resources.jointjs.com/tutorial/angular
